MapboxNavigation/CPTrip.swift

```diff
@@ -36,7 +36,7 @@ extension CPTrip {
             let routeChoice = CPRouteChoice(summaryVariants: summaryVariants,
                                             additionalInformationVariants: [route.description],
                                             selectionSummaryVariants: [route.description])
-            let info: (Route, RouteOptions) = (route: route, options: routeOptions)
+            let info: (Route, Int, RouteOptions) = (route: route, routeIndex: routeIndex, options: routeOptions)
             routeChoice.userInfo = info
```

Comment:
It’s frustrating that the only way to associate arbitrary data with UI elements that CarPlay controls is to stuff it in an untyped property and cast it later. As tail work, it might be better to define a set of dictionary keys and set this property to a dictionary, similar to when posting a Notification.